src/main/java/gameframework/drawing/DrawableImage.java

```diff
@@ -17,7 +17,14 @@ public DrawableImage(URL imageUrl, GameCanvas gameCanvas) {
 		}
 		handleImage(imageUrl);
 	}
-
+	
+	/**
+	 * Use a MediaTracker to load the image in the canvas, and throw an
+	 * exception if there is a problem
+	 * 
+	 * @param imageUrl
+	 *            is the path to the image
+	 */
```

Comment:
this comment has nothing to do with this pull request. Please open a separate one.